### DISCUSSION TITLE
关于资源包不兼容的问题

### DISCUSSION THREAD

**Discussion by HANTON114514** [Meloong-Git/PCL#588] [提问] [answered]
上次我下了个1.19.3的版本，他说那个资源包不兼容，所以我的皮肤片成了初始皮，连材质都变成旧版本的，就很离谱，而且启动器日志也没有下载新版资源文件，谁能帮帮我？
[Log3.txt](https://github.com/Hex-Dragon/PCL2/files/10218989/Log3.txt)
![屏幕截图 2022-12-13 224325](https://user-images.githubusercontent.com/120468922/207364162-08a6bfa3-393c-415c-988b-03b2c28ff457.jpg)

**Comment by WForst-Breeze** [comment]


**Reply by HANTON114514** [reply]
撒意思（疑问）

**Reply by HANTON114514** [reply]
我的皮肤真的变成那9个皮肤中的一个了！

**Comment by HANTON114514** [comment]
我照做了，可......
(西巴，mc截图也出问题了）
还是原始皮肤

**Comment by HANTON114514** [comment]
（截图没问题）
![2022-12-14_08 06 49](https://user-images.githubusercontent.com/120468922/207472400-76c3d271-db9c-494e-a228-8a4d5edcbec3.png)
s是这玩意儿

**Reply by Michael66625** [reply]
我的离线模式和非离线模式都有这个问题，均是官方默认皮肤中的一个。

**Comment by WForst-Breeze** [comment, marked as answer]
刚刚我也去试了一下，貌似真的没法更改离线模式皮肤……
反馈已提交，见 #590 #591

**Comment by HANTON114514** [comment]
啊西巴，谢谢嗷，不过这得跟龙腾反映一下
